src/languages/singlestoredb/singlestoredb.formatter.ts

```diff
@@ -0,0 +1,296 @@
+  'CREATE TABLESPACE',
+  'CREATE TRIGGER',
+  'CREATE USER',
+  'CREATE VIEW',
+  'DEALLOCATE PREPARE',
+  'DESCRIBE',
+  'DO',
+  'DROP DATABASE',
+  'DROP EVENT',
+  'DROP FUNCTION',
+  'DROP FUNCTION',
+  'DROP INDEX',
+  'DROP LOGFILE GROUP',
+  'DROP PROCEDURE',
+  'DROP RESOURCE GROUP',
+  'DROP ROLE',
+  'DROP SERVER',
+  'DROP SPATIAL REFERENCE SYSTEM',
+  'DROP TABLESPACE',
+  'DROP TRIGGER',
+  'DROP USER',
+  'DROP VIEW',
+  'EXECUTE',
+  'EXPLAIN',
+  'FLUSH',
+  'FORCE',
+  'GRANT',
+  'HANDLER',
+  'HELP',
+  'IMPORT TABLE',
+  'INSERT',
+  'INSTALL COMPONENT',
+  'INSTALL PLUGIN',
+  'KILL',
+  'LOAD DATA',
+  'LOAD INDEX INTO CACHE',
+  'LOAD XML',
+  'LOCK INSTANCE FOR BACKUP',
+  'LOCK TABLES',
+  'MASTER_POS_WAIT',
+  'OPTIMIZE TABLE',
+  'PREPARE',
+  'PURGE BINARY LOGS',
+  'REBALANCE PARTITIONS',
+  'RELEASE SAVEPOINT',
+  'RENAME TABLE',
+  'RENAME USER',
+  'REMOVE AGGREGATOR',
+  'REMOVE LEAF',
+  'REPAIR TABLE',
+  'REPLACE',
+  'REPLICATE DATABASE',
+  'RESET',
+  'RESET MASTER',
+  'RESET PERSIST',
+  'RESET REPLICA',
+  'RESET SLAVE',
+  'RESTART',
+  'RESTORE DATABASE',
+  'RESTORE REDUNDANCY',
+  'REVOKE',
+  'ROLLBACK',
+  'ROLLBACK TO SAVEPOINT',
+  'SAVEPOINT',
+  'SELECT',
+  'SET CHARACTER SET',
+  'SET DEFAULT ROLE',
+  'SET NAMES',
+  'SET PASSWORD',
+  'SET RESOURCE GROUP',
+  'SET ROLE',
+  'SET TRANSACTION',
+  'SHOW',
+  'SHOW BINARY LOGS',
+  'SHOW BINLOG EVENTS',
+  'SHOW CHARACTER SET',
+  'SHOW COLLATION',
+  'SHOW COLUMNS',
+  'SHOW CREATE DATABASE',
+  'SHOW CREATE EVENT',
+  'SHOW CREATE FUNCTION',
+  'SHOW CREATE PROCEDURE',
+  'SHOW CREATE TABLE',
+  'SHOW CREATE TRIGGER',
+  'SHOW CREATE USER',
+  'SHOW CREATE VIEW',
+  'SHOW DATABASES',
+  'SHOW ENGINE',
+  'SHOW ENGINES',
+  'SHOW ERRORS',
+  'SHOW EVENTS',
+  'SHOW FUNCTION CODE',
+  'SHOW FUNCTION STATUS',
+  'SHOW GRANTS',
+  'SHOW INDEX',
+  'SHOW MASTER STATUS',
+  'SHOW OPEN TABLES',
+  'SHOW PLUGINS',
+  'SHOW PRIVILEGES',
+  'SHOW PROCEDURE CODE',
+  'SHOW PROCEDURE STATUS',
+  'SHOW PROCESSLIST',
+  'SHOW PROFILE',
+  'SHOW PROFILES',
+  'SHOW RELAYLOG EVENTS',
+  'SHOW REPLICA STATUS',
+  'SHOW REPLICAS',
+  'SHOW SLAVE',
+  'SHOW SLAVE HOSTS',
+  'SHOW STATUS',
+  'SHOW TABLE STATUS',
+  'SHOW TABLES',
+  'SHOW TRIGGERS',
+  'SHOW VARIABLES',
+  'SHOW WARNINGS',
+  'SHUTDOWN',
+  'SOURCE_POS_WAIT',
+  'START GROUP_REPLICATION',
+  'START REPLICA',
+  'START SLAVE',
+  'START TRANSACTION',
+  'STOP GROUP_REPLICATION',
+  'STOP REPLICA',
+  'STOP REPLICATING',
+  'STOP SLAVE',
+  'TABLE',
+  'TRUNCATE TABLE',
+  'UNINSTALL COMPONENT',
+  'UNINSTALL PLUGIN',
+  'UNLOCK INSTANCE',
+  'UNLOCK TABLES',
+  'USE',
+  'WITH',
+  'XA',
+  // flow control
+  // 'IF',
+  'ITERATE',
+  'LEAVE',
+  'LOOP',
+  'REPEAT',
+  'RETURN',
+  'WHILE',
+  // other
+  'ADD',
+  'ALTER COLUMN',
+  'FROM',
+  'GROUP BY',
+  'HAVING',
+  'INSERT INTO',
+  'LIMIT',
+  'OFFSET',
+  'ORDER BY',
+  'WHERE',
+  'WINDOW',
+  'PARTITION BY',
```

Comment:
The `reservedCommands` list contains several duplicates like `SELECT`, `FROM`, `LIMIT`, `GROUP BY`, `INSERT` ...

Reply:
Okay, so if I understand it correctly, you are saying that if I have a definition like `INSERT [INTO]`, then it is okay to delete the `INSERT INTO` definition later in the `reservedCommands` section.

Reply:
Yep, you're correct.